web/src/components/molecules/Asset/Asset/AssetBody/index.tsx

```diff
@@ -43,17 +40,17 @@ const AssetBody: React.FC<Props> = ({
   handleModalCancel,
   handleFullScreen,
 }) => {
-  const { fileName, createdAt, createdBy } = asset;
   const { svgRender, handleCodeSourceClick, handleRenderClick } = useHooks();
-  const formattedCreatedAt = dateTimeFormat(createdAt);
-  const displayUnzipFileList = selectedPreviewType === PreviewType.ZIP;
+  const formattedCreatedAt = dateTimeFormat(asset?.createdAt);
```

Comment:
```suggestion
  const formattedCreatedAt = dateTimeFormat(asset.createdAt);
```
asset is always required

Reply:
Please update everywhere where its `asset?.XASLKJ`